pkg/providers/vsphere/vsphere.go

```diff
@@ -1089,7 +1081,7 @@ func BuildTemplateMap(clusterSpec *cluster.Spec, datacenterSpec v1alpha1.VSphere
 
 	if clusterSpec.Spec.ProxyConfiguration != nil {
 		values["proxyConfig"] = true
-		var noProxyList []string
+		noProxyList := make([]string, 4)
```

Comment:
I forget, does this make the capacity 4 or the size 4 with 4 `nil` appended to it? I ran into this problem before so wanted to make sure.